alby.go

```diff
@@ -404,7 +404,7 @@ func (svc *AlbyOAuthService) ListTransactions(ctx context.Context, senderPubkey
 		endpoint += "/outgoing"
 	}
 
-	requestUrl := fmt.Sprintf("%s%s?%s", svc.cfg.AlbyAPIURL, endpoint, urlParams.Encode())
+	requestUrl := fmt.Sprintf("%s%s?%s", svc.cfg.OAuthAPIURL, endpoint, urlParams.Encode())
```

Comment:
careful this would break the current deployment.
can you take care that the deployment is properly configured.

Reply:
Will take care of it once this is merged 👍